datasets/_externals/wikidata.yml

```diff
@@ -653,6 +653,8 @@ lookups:
           - Electorate of Mainz
           - Kingdom of Numidia
           - Spanish protectorate in Morocco
+          - State of Judaea
+          - Uyghur
```

Comment:
Removing Uyghur since in this case they referred to Uyghur language for citizenship. I removed the invalid statement in wikidata instead.

We might want to map Uyghur to China geographically, instead of nothing. But we can unpack that if it comes up again.
```suggestion
```